src/ethereum/api/Bridge.ts

```diff
@@ -22,6 +24,28 @@ export class Bridge {
       collections.push(collection)
     }
 
+    const remoteCollectionCreators = remoteCollections.map((collection) =>
+      getAddress(collection.creator)
```

Comment:
`getAddress` is used to validate an address and also to checksum it. Here, I checksum it because creators are being stored checksummed. What is really weird is why I am looking for creators instead of contract addresses.

I changed it to contract address.